crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -461,8 +461,20 @@ impl RoomEventCacheInner {
                 })
                 .await?;
 
+            if prev_batch.is_some() && !all_duplicates {
```

Comment:
Maybe explains this is how we compute the `limited` flag here?

Reply:
I am not sure what you mean, since we're not computing the `limited` flag anymore? :thinking:

Reply:
Got confused during the review. Marked as solved.